lib/cylc/scheduler.py

```diff
@@ -714,7 +740,7 @@ def configure_suite(self, reconfigure=False):
             else:
                 # Remove database created by previous runs
                 if os.path.isdir(pri_db_path):
-                    shutil.rmtree(pri_db_path)
+                    rmtree(pri_db_path)
```

Comment:
It think it was probably an overly cautious test. No, it should never happen in real life (but one can never be too careful when there are users!)